less/classbxr.less

```diff
@@ -12,9 +12,9 @@
     background-color: white;
     color: cornflowerblue;
     position: absolute;
-    top: -.5em;
+    top: -1.5ex;
     white-space: nowrap;
-    font-family: sans-serif;
+    font-family: monospace;
```

Comment:
Ok, I'll change it back. I though that since it's code then `monospace` it's more appropriate.